src/Symfony/Component/Validator/Context/LegacyExecutionContextFactory.php

```diff
@@ -11,6 +11,8 @@
 
 namespace Symfony\Component\Validator\Context;
 
+trigger_error('The '.__NAMESPACE__.'\LegacyExecutionContextFactory is deprecated since version 2.5 and will be removed in 3.0.', E_USER_DEPRECATED);
```

Comment:
should be removed, because Symfony uses this class all the time to be able to use the BC layer.